docs/guides/scaling.rst

```diff
@@ -0,0 +1,122 @@
+.. include:: ../refs.rst
```

Comment:
On one hand the existing books are guides, but on the other hand their scope and size is much different than the scaling guide. I've moved the scaling doc to `docs/HowTos` now (shall we rename it to "How do I setup PCP at scale?" instead of the current "Scaling Guidelines"?).

fwiw, for the future mini guides/tutorials (e.g. for pmie etc.), I suggest we take https://kubernetes.io/docs/tasks/run-application/run-single-instance-stateful-application/ as an example (the kubernetes docs are one of the best docs I've seen so far). They're calling it tasks (our task could be "Setup a custom pmie(1) rule").